pkg/auditlog/storage/v2/admin_audit_log.go

```diff
@@ -103,22 +100,18 @@ func (s *adminAuditLogStorage) ListAdminAuditLogs(
 	whereSQL, whereArgs := mysql.ConstructWhereSQLString(whereParts)
 	orderBySQL := mysql.ConstructOrderBySQLString(orders)
 	limitOffsetSQL := mysql.ConstructLimitOffsetSQLString(limit, offset)
-	query := fmt.Sprintf(`
```

Comment:
To maintain consistency with the other queries, we should do the following:

#### Storage
https://github.com/bucketeer-io/bucketeer/blob/main/pkg/account/storage/v2/account.go#L243-L249

#### Query file
https://github.com/bucketeer-io/bucketeer/blob/main/pkg/account/storage/v2/sql/account_v2/select_accounts_v2.sql

If we decide to write the query using `strings.Builder`, we should do the same for all queries.
Is there a reason for changing the current format?

Reply:
I fixed not using `strings.Builder`.
https://github.com/bucketeer-io/bucketeer/pull/892/commits/7b0c92a29d0c06b5a9814c571d1386f4dcbfc555